CONTRIBUTING.md

```diff
@@ -61,3 +61,5 @@ TomaszLukawskiSam
 
 * PR can be merged by PR owner or TAU reviewer if at least one TAU Reviewer approved and verified it.
 * TAU Reviewers can not approve their own PR.
+* When merging **standard PR use "Rebase and merge"** option in order to have straight git history
```

Comment:
How about the "Squash and merge" option?
Sometimes the merged PR remained  "Merge pull request #000  from Samsung/xxx".
I think it doesn't good look.

Reply:
"Rebase and merge" option will all to avoid such "merge" commit.
It puts the commit on the top of history so history is straight and there is no "merge" in history.
In case of "squash and merge" few commits which are part of PR are squashed into one and merged. This is not good for clear history of changes. Sometimes we may want to split some change into few commits intentionally. "squash and merge" will destroy our intentions

Reply:
@lslachciak 
It is not good for us to have a merge or pull history. 
and developers always rebase themselves before merging commits. 
this is because the code is not split and merged properly. 
If it is necessary to 'rebase and merge', it proceeds, but I think it is unnecessary because git proceeds with rebase. (This is my subjective thought.)